src/SpeechRecognition.js

```diff
@@ -71,7 +71,7 @@ export default function SpeechRecognition(options) {
         listening = false
         if (pauseAfterDisconnect) {
           this.setState({ listening })
-        } else {
+        } else if (typeof options.continuous === 'undefined') {
```

Comment:
`SpeechRecognition` should resume listening on a disconnect/end-of-speech if `options.continuous` is `undefined` _or_ `true`:

```suggestion
        } else if (options.continuous !== false) {
```

Alternatively, you could inspect the recognition object directly:

```suggestion
        } else if (recognition && recognition.continuous) {
```